waspc/waspls/src/Wasp/LSP/Util.hs

```diff
@@ -40,3 +47,17 @@ anyP preds x = any ($ x) preds
 -- | Lift a 'Maybe' into a 'MaybeT' monad transformer.
 hoistMaybe :: Applicative m => Maybe a -> MaybeT m a
 hoistMaybe = MaybeT . pure
+
+stripProperPrefix :: SP.Path' SP.Abs (SP.Dir a) -> SP.Path' SP.Abs (SP.File b) -> Maybe (SP.Path' (SP.Rel a) (SP.File b))
+stripProperPrefix base file =
+  SP.fromPathRelFile
+    <$> P.stripProperPrefix (SP.toPathAbsDir base) (SP.toPathAbsFile file)
```

Comment:
Let's put this one into Wasp.Util.StrongPath, and you can also add it to an issue we have for this on StrongPath repo.